dev_guide/deployments.adoc

```diff
@@ -670,3 +670,39 @@ $ oc scale dc frontend --replicas=3
 
 The number of replicas eventually propagates to the desired and current
 state of the deployment configured by the deployment configuration `frontend`.
+
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+ifdef::openshift-enterprise,openshift-origin[]
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+endif::[]
+
+OpenShift cluster administrators can set the default node selector for your project in order to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a pod configuration to restrict nodes even further.
+
+To add a node selector when creating a pod, edit the pod configuration, and add the `nodeSelector` value. This can be added to a single pod configuration, or in a pod template:
+
+====
+----
+...
+nodeSelector:
+  disktype: ssd
```

Comment:
If we're not going to link to upstream doc, then this should have a little more context as they'd have to look at the API spec to figure out where exactly this goes. I suggest the example look like:

```
apiVersion: v1
kind: Pod
spec:
  nodeSelector:
    disktype: ssd
...
```